Flock vulnerability false positive (#13827)
fleetdm/fleet: docs/Using Fleet/Vulnerability-Processing.md

> ### Advanced Configuration

Reviewer: I realize this was already like this, but gonna suggest fixing the heading capitalization while we're here:
```suggestion
### Advanced configuration
```

  }
]
```

This translation rule maps installed homebrew packages with a name containing `ruby` with a matching
CPE using `product == "ruby" AND vendor == "ruby-lang"`

_sample CPE:_

> ```text

Author: Thanks!  I ended up adding the syntax highlighting to all code blocks and some more formatting changes based on my [linter](https://github.com/DavidAnson/markdownlint)

### Matching a CPE to a CVE

Once we have a good CPE, we can match it against the CVE database. We download the data streams
locally and match each CPE to the whole list. The matching is done using the [nvdtools
implementation](https://github.com/facebookincubator/nvdtools).

### False Positive Cleanup

> False positive entries are removed during vulnerability processing if the entry hasn't been updated in 2hrs

Reviewer 2: 2 hours by default, it's actually `2 * periodicity`.

| Name        | Type          | Description              |
| ---         | ---           | ---                      |
| `product`   | array[string] | The CPE product.        |
| `vendor`    | array[string] | The CPE vendor.          |
| `target_sw` | array[string] | The CPE target software. |
| `skip`      | bool          | If true, matched software will be skipped from the NVD vulnerability scanning process |

### Matching a CPE to a CVE

Once we have a good CPE, we can match it against the CVE database. We download the data streams
locally and match each CPE to the whole list. The matching is done using the [nvdtools
implementation](https://github.com/facebookincubator/nvdtools).

### False positive cleanup

> if the span of time since the entry was updated is greater than 2x the (configured periodicity)[https://fleetdm.com/docs/configuration/fleet-server-configuration#periodicity].

Reviewer: Oops! My suggested change cut off the 1st part of the sentence here 🤦‍♀️  I'll go ahead and commit this update now:
```suggestion
False positive entries are removed during vulnerability processing if the span of time since the entry was updated is greater than 2x the (configured periodicity)[https://fleetdm.com/docs/configuration/fleet-server-configuration#periodicity].
```

> False positive entries are removed during vulnerability processing if the span of time since the entry was updated is greater than 2x the (configured periodicity)[https://fleetdm.com/docs/configuration/fleet-server-configuration#periodicity].

Reviewer: ...whoops, reversed the `()` and the `[]`, that's why the link didn't work. Will commit this change momentarily:
```suggestion
False positive entries are removed during vulnerability processing if the span of time since the entry was updated is greater than 2x the [configured periodicity](https://fleetdm.com/docs/configuration/fleet-server-configuration#periodicity).
```

| Name        | Type          | Description              |
| ---         | ---           | ---                      |
| `product`   | array[string] | The CPE product.        |
| `vendor`    | array[string] | The CPE vendor.          |
| `target_sw` | array[string] | The CPE target software. |
| `skip`      | bool          | If true, matched software will be skipped from the NVD vulnerability scanning process |

### Matching a CPE to a CVE

Once we have a good CPE, we can match it against the CVE database. We download the data streams
locally and match each CPE to the whole list. The matching is done using the [nvdtools
implementation](https://github.com/facebookincubator/nvdtools).

> ### False Positive Cleanup

Reviewer: [Sentence-case](https://fleetdm.com/handbook/company/communications#using-sentence-case-and-capitalization) heading:
```suggestion
### False positive cleanup
```

> False positive entries are removed during vulnerability processing if the entry hasn't been updated
> in 2 * (periodicity)[https://fleetdm.com/docs/configuration/fleet-server-configuration#periodicity]

Reviewer: Wondering if there's a way to make this wording a bit clearer... what do you think of:
```suggestion
if the span of time since the entry was updated is greater than 2x the (configured periodicity)[https://fleetdm.com/docs/configuration/fleet-server-configuration#periodicity].
```
